perses/app/setup_relative_calculation.py

```diff
@@ -468,6 +468,8 @@ def run_setup(setup_options):
         _logger.info(f"\tno nonequilibrium detected.")
         n_states = setup_options['n_states']
         _logger.info(f"\tn_states: {n_states}")
+        n_replicas = setup_options['n_states']
```

Comment:
Should this be `n_replicas = setup_options['n_replicas']` if you want to have different numbers of states and replicas? Or are you trying to change it so that the default behaviour is n_replicas = n_states?

Reply:
my bad! my intention was exactly `n_replicas = setup_options['n_replicas']`. I'll fix it now. Thanks!